compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx

```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
+<Message type="important">
 
-  ```
-  Create a new baremetal server. Once the server is created, you probably want to install an OS.
-
-  USAGE:
-    scw baremetal server create [arg=value ...]
-
-  EXAMPLES:
-    Create instance
-      scw baremetal server create
-
-    Create a GP-BM1-M instance, give it a name and add tags
-      scw baremetal server create name=foo tags.0=prod tags.1=blue type=GP-BM1-M
-
-  ARGS:
-    [project-id]                    Project ID to use. If none is passed the default project ID will be used
-    name=<generated>                Name of the server (≠hostname)
-    [description]                   Description associated to the server, max 255 characters
-    [type=GP-BM1-S]                 Server commercial type (GP-BM1-L | GP-BM1-M | GP-BM1-S | HC-BM1-L | HC-BM1-S | HM-BM1-XL | HM-BM1-M)
-    [tags.{index}]                  Tags to associate to the server
-    [install.os-id]                 
-    [install.hostname]              
-    [install.ssh-key-ids.{index}]   
-    [organization-id]               Organization ID to use. If none is passed the default organization ID will be used
-    [zone=fr-par-1]                 Zone to target. If none is passed will use default zone from the config (fr-par-2)
-
-  FLAGS:
-    -h, --help   help for create
-    -w, --wait   wait until the server is ready
-
-  GLOBAL FLAGS:
-    -c, --config string    The path to the config file
-    -D, --debug            Enable debug mode
-    -o, --output string    Output format: json or human, see 'scw help output' for more info (default "human")
-    -p, --profile string   The config profile to use
-
-  SEE ALSO:
-    # List os
-    scw baremetal os list
-
-    # Install an OS on your server
-    scw baremetal server install
-  ```
+  - Bare Metal servers are only available in the `fr-par-2` zone at the moment. 
+  - The available server types are the following: `GP-BM1-L` `GP-BM1-M` `GP-BM1-S` `HC-BM1-L` `HC-BM1-S` `HM-BM1-XL` `HM-BM1-M`.
 
-  In `ARGS` you see all the options for creating your Bare Metal server, including the different types of server available (GP-BML-L, GP-BM1-M etc).  
+</Message>
 
-  <Message type="Note">
-  
-  The help info for the `zone` argument says that the default zone is `fr-par-2`. In fact, `fr-par-2` is the only zone in which Bare Metal servers are available at the moment, but it is not necessarily the default zone in your configuration. To create a Bare Metal server, you need to either:
-  a) change the default zone in your configuration file (check the location of this file in the output of sw init and change the value for `default zone` to `fr-par-2`), or
-  b) explicitly target the fr-par-2 zone in the commands to create and manage your server.
 
-  In this tutorial, we’ll target the zone explicitly in our commands.
+1. Type the following command in your terminal to create your Bare Metal Server:
 
-  </Message>
+  `scw baremetal server create name=name-of-your-server type=GP-BM1-L zone=fr-par-2`
```

Comment:
```suggestion
  ```
  scw baremetal server create name=name-of-your-server type=GP-BM1-L zone=fr-par-2
  ```
```